docs/src/pages/components/use-media-query/use-media-query.md

```diff
@@ -49,6 +39,12 @@ function MyComponent() {
 
 {{"demo": "pages/components/use-media-query/ThemeHelper.js"}}
 
+## Using JavaScript syntax
+
+[json2mq](https://github.com/akiran/json2mq) is used to generate media query string from a JavaScript object.
+
+{{"demo": "pages/components/use-media-query/JavaScriptMedia.js", "defaultCodeOpen": true}}
```

Comment:
I will come back to it in two months, I will use the Google Analytics events to better answer this question: Should we have json2mq built-in.